### DISCUSSION TITLE
Deno support

### DISCUSSION THREAD

**Discussion by aadamsx** [tailwindlabs/tailwindcss#5710] [Ideas]
### What version of Tailwind CSS are you using?

2.0.4

### What version of Node.js are you using?

N/A

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### Reproduction repository

N/A

Please add Deno support to Tailwind, I'm using Deno/Alephjs (React) and would really like to use Tailwind as I do in my Node/Nextjs (React) projects: 

https://github.com/alephjs/aleph.js/issues/42#issuecomment-806056884

https://github.com/postui/esm.sh/issues/10#issue-779571089

ref: https://github.com/alephjs/aleph.js/issues/42#issuecomment-806056884

**Comment by Xeevis** [comment]
With recently published [TailwindCSS Standalone CLI](https://tailwindcss.com/blog/standalone-cli) it's quite easy to use TailwindCSS with [`Deno.run`](https://deno.land/manual/examples/subprocess) with no Node whatsoever. To be able to use PostCSS plugins there are official modules for both [PostCSS ](https://deno.land/x/postcss) and for example [postcss-import](https://deno.land/x/postcss_import).

PostCSS needs to run first for imports to be inlined, resulting css can then be piped to TailwindCSS CLI.

```ts
const twprocess = Deno.run({
  cmd: [
    "cmd", "/C", "./tailwindcss-windows-x64", "-i", "app.pcss", "-o", "app.css", "-c", "tailwind.config.cjs",
  ]
});
await twprocess.status();
twprocess.close();
```

TailwindCSS Cli includes cssnano (use with `--minify`) and autoprefixer (enabled by default). But you can pipe stdout from CLI to esbuild or whatever for further processing.
